src/applications/check-in/components/AppointmentDisplay/CheckInButton.jsx

```diff
@@ -34,28 +34,38 @@ const CheckInButton = ({
     },
     [checkInWindowEnd, eventRecorder, getCurrentPageFromRouter, onClick],
   );
-
+  if (isCheckingIn) {
```

Comment:
Do you want to keep the comment here?

Reply:
yes, just a little documentation for why there are two button renders.